src/components/schemas/expense_policies.yaml

```diff
@@ -0,0 +1,654 @@
+      allOf:
+        - $ref: './fields.yaml#/timestamptz_pst'
+      description: |
+        Expense having spent date lesser than this date will violate policy.
+    expense_payment_modes:
+      allOf:
+        - $ref: ./fields.yaml#/minimal_payment_modes
+      description: |
+        Payment mode for expense to be targeted in the policy.
+    expense_payment_modes_op:
+      $ref: ./fields.yaml#/in_operator
+    expense_without_ccc_match_allowed:
+      type: boolean
+      description: |
+        Is expense without Corporate Credit Card transaction match allowed.
+      example: True
+    expense_include_foreign_currency:
+      type: boolean
+      description: |
+        Should policy run on expenses created in foreign currency.
+      example: True
+    expense_flight_travel_class:
+      $ref: ./fields.yaml#/expense_policy_flight_travel_classes
+    expense_train_travel_class:
+      type: array
+      description: >
+        This field is applicable only for `Train` category.
+
+        Index 0 holds onward journey travel class, index 1 holds return journey
+        travel class.
+      items:
+        type: string
+      example:
+        - AC First Class
+        - Second Sitting
+    fiscal_year_start_month:
+      type: number
+      description: Month number of start of fiscal year.
+      example: 4
+      minimum: 1
+      maximum: 12
+    expense_vehicle_type:
+      allOf:
+        - $ref: ./fields.yaml#/vehicle_type
+      description: |
+        Vehicle type targeted by this policy.
+    expense_annual_distance_max:
+      type: number
+      example: 1200
+      description: |
+        Maximum travel distance allowed, above which policy will be violated. `expense_category_ids` must be set with mileage categories.
+    employee_level_ids:
+      type: array
+      description: |
+        Employee level ids targeted by this policy.
+      items:
+        type: string
+      example:
+        - lvl1sdfad
+        - lvl2sdfas
+    employee_level_ids_op:
+      $ref: ./fields.yaml#/in_operator
+    employee_custom_fields:
+      $ref: './fields.yaml#/custom_fields'
+    employee_custom_fields_op:
+      type: object
+      description: |
+        Exmployee custom field operator. <br>
+        field_name: operator <br>
+        Example -> gender: "IN" <br><br>
+        If field type is, then supported are: <br>
+        `Boolean`: "IS, NOT_IS" <br>
+        `Date`: "LESS_THAN_EQUAL_TO, GREATER_THAN_EQUAL_TO" <br>
+        `Location`: "IN, NOT_IN" <br>
+        `Number`: "LESS_THAN_EQUAL_TO, GREATER_THAN_EQUAL_TO" <br>
+        `String`: "CONTAINS, EQUALS" <br>
+        `Select`: "IN, NOT_IN"
+      properties:
+        field_name:
+          type: string
+    employee_department_ids:
+      type: array
+      description: |
+        Employee department ids targeted by this policy.
+      items:
+        type: string
+      example:
+        - dept1asdfa
+        - dept2adfad
+    employee_department_id_op:
```

Comment:
<some description><br/>
Note: This field is mandatory if the field `employee_department_ids` is present in the request payload/body.